### PR TITLE
build(client): Ignore bundle analysis files in prettier

### PR DESCRIPTION
**Pull request by tylerbutler**
These files are automatically generated at build time and should not be formatted with prettier.

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -31,6 +31,9 @@ packages/tools/fluid-runner/src/test/telemetryExpectedOutputs/
 # Generated type-tests
 **/*.generated.ts
 
+# Generated bundle analysis files
+**/bundleAnalysis/**
```
Comment (CraigMacomber) [inline .prettierignore:35]: is this really the right place to ignore them? THis seems like an unrelated packages ignore file

Comment (CraigMacomber) [reply .prettierignore:35]: Should be in examples/utils/bundle-size-tests right?

Comment (tylerbutler) [reply .prettierignore:35]: There's one ignore file for the client release group, so this is the right place. @Josmithr @jikim-msft Do we have any docs/rationale behind the single ignore file vs. individual ones in each package?

Comment (tylerbutler) [reply .prettierignore:35]: Oops, you're right. I am blind apparently.

Comment (tylerbutler) [reply .prettierignore:35]: No, I was correct originally... this is the root .prettierignore, which is correct for client.

Comment (Josmithr) [reply .prettierignore:35]: We do have some packages that declare their own `.prettierignore` files for situations like this. It's probably worth making one for that package, and updating the scripts to point to it (you can just remove the path argument from them, since same dir is the default place it looks).

Comment (Josmithr) [reply .prettierignore:35]: As for rationale, I see the root ignore file as the "default", and individual packages can override as needed. We should probably document that better - probably worth adding a comment to each of the release-group-roots' ignore file.

Comment (tylerbutler) [reply .prettierignore:35]: One problem is that the individual packages don't really "override" the default, do they? Does it look at both files (the one locally and the one in the root)?

**Review by jikim-msft — APPROVED** [review]

File: .prettierignore (hunk 1 above)
Comment (Josmithr) [reply .prettierignore:35]: That's a fair point. I actually don't know for sure what prettier will do as it descends the directory tree. It may or may not respect local configs instead of ancestor configs. It may also depend on whether or not you explicitly specify an ignore file as a script param. This could use some exploration.
